dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -147,8 +147,19 @@ func buildModules(srcPath, name, ufsType, moduleFlag, version string, modules ma
 		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
 			moduleMvnArgs = append(moduleMvnArgs, arg)
 		}
+		var versionMvnArg string
+		for _, arg := range strings.Split(moduleEntry.mavenArgs, " ") {
+                        if strings.Contains(arg, "ufs.hadoop.version") {
+				versionMvnArg = strings.Split(arg, "=")[1]
+			}
+                }
```

Comment:
indentation is off for the two closing braces

Reply:
fixed. golluxio uses tabs and alluxio uses spaces.